pagopa/pn-frontend
# Pn 2390 filtro notifiche comportamento non corretto se si pulisce un parametro in alcuni scenari

## Short description
Wrong behaviour of submit and reset buttons in filter notifications.

## List of changes proposed in this pull request
- Added conditional value assignment when values are null in datepickers

## How to test
Enter a date or date range and filter. Then delete date(s), the filter button should still be enabled
